yt/visualization/fixed_resolution.py

```diff
@@ -639,7 +639,6 @@ def _generate_image_and_mask(self, item) -> None:
                 self.bounds[5] - self.bounds[4],
             )
         )
-        depth = dd.depth[0] if dd.depth is not None else None
```

Comment:
removing this because the depth sanitization before `dd` creation takes care of this now.